spark/src/main/java/org/apache/iceberg/actions/ExpireSnapshotsAction.java

```diff
@@ -56,44 +36,15 @@
  * require a shuffle so parallelism can be controlled through spark.sql.shuffle.partitions. The expiration is done
  * locally using a direct call to RemoveSnapshots. The snapshot expiration will be fully committed before any deletes
  * are issued. Deletes are still performed locally after retrieving the results from the Spark executors.
+ *
+ * @deprecated since 0.12.0
```

Comment:
We should also tell the reader what to use instead and when to expect this to be removed. That also helps us find things that should be removed for a release by grepping for "will be removed in 0.13.0".

Reply:
Fixed.